sdk/python/scripts/generate_protos.py

```
@@ -0,0 +1,201 @@
+        "-m",
+        "grpc_tools.protoc",
+        f"-I{PROTO_INCLUDE}",
+        f"-I{_grpc_include()}",
+        f"--python_out={destination}",
+        f"--grpc_python_out={destination}",
+        *(str(path) for path in protos),
+    ]
+    environment = os.environ.copy()
+    environment.pop("PYTHONHOME", None)
+    environment.pop("PYTHONPATH", None)
+    result = subprocess.run(command, cwd=str(REPO_ROOT), env=environment,
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)